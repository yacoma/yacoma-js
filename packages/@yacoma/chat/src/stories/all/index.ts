export * from './Explorer'
// export * from './Header'
// export * from './Chat'
// export * from './People'
// export * from './ContactsWidget'
//
