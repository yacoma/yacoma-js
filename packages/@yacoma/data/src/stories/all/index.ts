export * from './ContactPrint'
export * from './Loading'
export * from './RegisterName'
export * from './RegisterVerify'
export * from './RegisterWords'
export * from './WordLogin'
