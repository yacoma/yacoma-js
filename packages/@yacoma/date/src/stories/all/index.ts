export * from './Date'
export * from './RelativeDate'
export * from './DatePicker'
