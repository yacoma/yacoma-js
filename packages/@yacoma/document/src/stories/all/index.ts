export * from './ExplorerWidget'
export * from './Explorer'
export * from './UserAccessView'
export * from './ItemTitle'
export * from './ItemIcon'
export * from './ItemDrag'
export * from './DocumentLink'
export * from './DocumentEditor'
