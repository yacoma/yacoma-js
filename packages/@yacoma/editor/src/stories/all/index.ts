export * from './Toolbox'
export * from './Editor'
export * from './CustomParagraph'
