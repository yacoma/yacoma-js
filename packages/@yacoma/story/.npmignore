/node_modules
/coverage
/setup
/src
/.cache
.gitignore
.npmignore
.prettierrc.json
.travis.yml
tsconfig.json
tslint.json
package-lock.json
yarn.lock