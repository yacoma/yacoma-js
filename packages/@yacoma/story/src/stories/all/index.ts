export * from './actions'
export * from './children'
export * from './props'
export * from './state'
export * from './storiesDefaults'
