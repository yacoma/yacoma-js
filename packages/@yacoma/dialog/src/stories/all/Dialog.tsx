import { action } from 'overmind'
import * as React from 'react'
import { build, settings } from '@yacoma/build'
import { TStories } from '@yacoma/story'
import { Dialog as component, dialog, DialogProps } from '../..'
import { DialogSettings } from '../../types'

const main = {
  name: 'main',
  settings: settings<DialogSettings>({
    dialog: {
      exclusive: {
        Exclusive: ({ children }) => <div>Exclusive: {children}</div>,
      },
      overlay: {
        Overlay: ({ children }) => <div>Overlay: {children}</div>,
      },
    },
  }),
}
const config = build(main)
  .using(dialog)
  .config()

export const modalStories: TStories<typeof config, DialogProps> = {
  component,
  config,
  name: 'Dialog (click to view)',
  stories: [
    {
      name: 'exclusive',
      props: {},
      titleClick: action(({ state }) => {
        if (state.dialog.show === 'Exclusive') {
          delete state.dialog.show
        } else {
          state.dialog.show = 'Exclusive'
        }
      }),
      children: <div>this is inside Dialog children</div>,
    },

    {
      name: 'overlay',
      props: {},
      titleClick: action(({ state }) => {
        if (state.dialog.show === 'Overlay') {
          delete state.dialog.show
        } else {
          state.dialog.show = 'Overlay'
        }
      }),
      children: <div>this is inside Dialog children</div>,
    },
  ],
}
