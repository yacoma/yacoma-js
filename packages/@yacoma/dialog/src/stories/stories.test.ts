import { testStories } from '@lucidogen/story'

import * as stories from './all'

testStories({}, stories)
