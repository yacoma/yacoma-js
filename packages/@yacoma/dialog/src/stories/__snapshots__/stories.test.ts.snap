// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog (click to view) should render exclusive 1`] = `
<div
  className="theme-Dialog--click-to-view--exclusive"
>
  <div
    className="sc-ifAKCX hYWxfb"
  >
    <div
      className="sc-htpNat hIwNRt"
    >
      <div
        className="sc-bxivhb glCqXO"
      >
        <div
          className="Title sc-bdVaJa dLCEWS"
          onClick={[Function]}
          style={
            Object {
              "cursor": "pointer",
            }
          }
        >
          exclusive
        </div>
        <div
          className="sc-iwsKbI dAfjtB"
        >
          &lt;Dialog&gt;...
          <div
            className="jsx"
          >
            &lt;Dialog&gt;
  &lt;div&gt;app content inside dialog chidren&lt;/div&gt;
&lt;/Dialog&gt;
          </div>
        </div>
      </div>
      <div
        className="sc-bwzfXH dNRbgB"
      >
        <div
          className="sc-dnqmqq bYUdBm"
        >
          <div>
            app content inside dialog chidren
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Dialog (click to view) should render overlay 1`] = `
<div
  className="theme-Dialog--click-to-view--overlay"
>
  <div
    className="sc-ifAKCX hYWxfb"
  >
    <div
      className="sc-htpNat hIwNRt"
    >
      <div
        className="sc-bxivhb glCqXO"
      >
        <div
          className="Title sc-bdVaJa dLCEWS"
          onClick={[Function]}
          style={
            Object {
              "cursor": "pointer",
            }
          }
        >
          overlay
        </div>
        <div
          className="sc-gqjmRU dHWrsZ"
        >
          &lt;Dialog&gt;...
          <div
            className="jsx"
          >
            &lt;Dialog&gt;
  &lt;div&gt;app content inside dialog chidren&lt;/div&gt;
&lt;/Dialog&gt;
          </div>
        </div>
      </div>
      <div
        className="sc-bwzfXH dNRbgB"
      >
        <div
          className="sc-gZMcBi dfNziT"
        >
          <div>
            app content inside dialog chidren
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
